_security-plugin/index.md

```diff
@@ -8,20 +8,66 @@ redirect_from:
-
-The security plugin has several default users, roles, action groups, permissions, and settings for OpenSearch Dashboards that use `kibana` in their names. We will change these names in a future release.
-{: .note }
-
-Feature | Description
-:--- | :---
-Node-to-node encryption | Encrypts traffic between nodes in the OpenSearch cluster.
-HTTP basic authentication | A simple authentication method that includes a user name and password as part of the HTTP request.
-Support for Active Directory, LDAP, Kerberos, SAML, and OpenID Connect | Use existing, industry-standard infrastructure to authenticate users, or create new users in the internal user database.
-Role-based access control | Roles define the actions that users can perform: the data they can read, the cluster settings they can modify, the indices to which they can write, and so on. Roles are reusable across users, and users can have multiple roles.
-Index-level, document-level, and field-level security | Restrict access to entire indices, certain documents within an index, or certain fields within documents.
-Audit logging | These logs let you track access to your OpenSearch cluster and are useful for compliance purposes or after unintended data exposure.
-Cross-cluster search | Use a coordinating cluster to securely send search requests to remote clusters.
-OpenSearch Dashboards multi-tenancy | Create shared (or private) spaces for visualizations and dashboards.
+# About Security for OpenSearch
+
+Security for OpenSearch is built around four main features that work together to safeguard data and track activity within the cluster. Separately, these features are:
+
+* Encryption
+* Authentication
+* Access control (authorization)
+* Audit logging and compliance
+
+Used together they provide effective protection of sensitive information by putting the information behind multiple layers of defense and granting or restricting access to information at different levels in the OpenSearch data structure. Most implementations use a combination of options for these features to meet unique and specific security needs.
+
+## Features at a glance
+
+The following topics provide a general description of the features that define security in OpenSearch.
+
+### Encryption
+
+Encryption protects both data at rest and in transit. At rest, encryption secures sensitive information that’s kept as stored data. Some examples of stored data include indexes, logs, swap files, automated snapshots, and all data in the application directory. 
+
+Encryption in transit encrypts information moving to, from, and within the cluster. OpenSearch uses the Transport Layer Security (TLS) protocol, which covers both client-to-node encryption (the REST layer) and node-to-node encryption (the transport layer). This combination of in-transit encryption guarantees that both requests to OpenSearch and the movement of information among different nodes is safe from tampering.
```

Comment:
I spelled out TLS mainly because I'm still looking it up myself. But I'm not attached. Will have either of the other two team members contribute an opinion.
RE: Guarantee. Would not have known that. Thanks.

Reply:
See "Don't spell out common acronyms" here: https://alpha.www.docs.aws.a2z.com/awsstyleguide/latest/styleguide/acronyms.html.

Reply:
Hmm, I guess I'll leave them spelled out for now, since it's the overview page. But only here. Thanks.